yaci-indexer/pom.xml

```diff
@@ -19,11 +19,6 @@
         <yaci-store.version>0.1.0-rc2-323a1a4-SNAPSHOT</yaci-store.version>
         <sonar.exclusions>src/main/java/org/cardanofoundation/rosetta/yaciindexer/stores/txsize/model/*</sonar.exclusions>
         <version.spotless-maven-plugin>2.43.0</version.spotless-maven-plugin>
-        <test.exclusions>
-            **/model/**,
-            **/yaciindexer/config/**,
```

Comment:
Don't we need to exclude this? Because we don't have tests for that and don't want to write tests for this?

Reply:
It's already added to base pom.xml 
` **/yaciindexer/**`

Reply:
now we have a lot of uncovered classes in sonar because of this reason
![image](https://github.com/cardano-foundation/cardano-rosetta-java/assets/46871134/27f8fee3-5eb6-4834-99b6-4f519f818dce)

Reply:
Ah ok!